.travis.yml

```diff
@@ -33,8 +30,9 @@ before_script:
 
 # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/recipe-cms:1.0.x-dev
+  - composer require --no-update silverstripe/recipe-cms:2.x-dev
   - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - composer require --prefer-source silverstripe/framework 5.x-dev
```

Comment:
Shouldn't this be before the `composer install`, and also have --no-update?

Reply:
I don't know - can you determine in whether to install from source or dist before you run a "install from dist" command? I expected that this flag would be negated by the use of `--no-update`

Reply:
Sorry, turns out it didn't work. `Nothing to install or update`